README.md

```diff
@@ -18,26 +16,52 @@ If you want to test the apps interaction with third party services (e.g. GitHub
 and GOV.UK Notify) you'll need to set up some secrets in environment variables.
 The easiest way to do this in development is using a `.env` file (see [.env.example](.env.example)):
 
-```
+```shell
 cp .env.example .env
 $EDITOR .env
 ```
 
 Run the tests with:
 
-```
+```shell
 bundle exec rails test
 ```
 
 To sign in as a development user, visit <http://localhost:3000/dev-login> (to try different email addresses, you can provide a `email` parameter). If you want to test with real Google SSO, you can [create an application in the Google Cloud Console](https://console.developers.google.com/apis/credentials).
 
-Deploying to PaaS
------------------
+## Building the Docker image
+
+In order to move this app away from GOV.UK PaaS, we have containerised the 
+application in order to be able to run locally and also deploy to AWS
+App Runner. The app can be accessed on port 3000.
+
+To build and run the Docker image:
+
+```shell
+docker build -t request-an-aws-account .
+docker run -p 3000:3000 -t request-an-aws-account
+```
+
+## Deploying through GitHub Actions
+
+We use a GitHub Action to automate the building and deploying of our Docker image to GHCR. This workflow runs every time a PR is raised to the branch called 'main'. It uses the `docker/login-action`, `docker/metadata-action`, and `docker/build-push-action`.
+
+You can view the workflow in action [here](https://github.com/alphagov/re-request-an-aws-account/actions/runs/7816113717).
+
+The workflow utilizes the following GitHub Actions:
+[`docker/login-action`](https://github.com/docker/login-action) for logging in to Docker.
+[`docker/metadata-action`](https://github.com/docker/metadata-action) for generating Docker image metadata.
+[`docker/build-push-action`](https://github.com/docker/build-push-action) for building and pushing the Docker image.
```

Comment:
Can you format this as a list of 3 items? Add a blank line between 51 and 52, then start the next 3 lines with a hyphen and a space.